ui/page/channel/view.jsx

```diff
@@ -19,7 +19,6 @@ import ChannelEdit from 'component/channelEdit';
 import classnames from 'classnames';
 import HelpLink from 'component/common/help-link';
 import ClaimSupportButton from 'component/claimSupportButton';
-import ChannelStakedIndicator from 'component/channelStakedIndicator';
 import ClaimMenuList from 'component/claimMenuList';
```

Comment:
In this zoom setting, the claim context menu shifts/scrolls the page when clicked to make the popup fit the screen.
- The shift is quite jarring
- But more importantly (typically on the right-most tile), I often ended up selecting one of the popup items (typically Support) ... i.e. clicking on the context menu showed the Support Modal instead context menu.

The current odysee doesn't shift but simply cuts off by the viewport ... that sucks too, but better than shifting?

![image](https://user-images.githubusercontent.com/64950861/149172781-728f307a-9881-40b1-8ece-f01528979cb1.png)

Reply:
At 125% zoom, correct? I'm not quite sure what is causing the issue but I'll look into it. Thanks!

However, I was already thinking about adding a zoom feature to the UI itself. At least for the tiles. Kinda like the one Plex has:
![image](https://user-images.githubusercontent.com/94839278/149176014-30744d70-a4d5-423e-bc2c-abd5d4d22543.png)
It would allow you to manually change the size of the previews and it would also make the zoom feature of the browser obsolete in most cases.

Reply:
Looks like this is the reason why the LBRY app and the current version of Odysee have a menu animation. DispatchCommit was the one who came up with this solution about a year ago: https://github.com/reach/reach-ui/issues/563#issuecomment-768891245

I'll try to find a better solution, but I don't think it's necessarily a priority in this particular PR since this only seems to happen on desktops with the browser zoom enabled. I can't replicate this on lower resolutions with the same 4 tiles per row layout.

Reply:
Was hitting this a few times today on 100% zoom on a 3840 * 1080 resolution.

https://user-images.githubusercontent.com/64950861/150523761-d6590391-5538-4ca8-adf0-2ee818b4aebf.mp4

Reply:
I've added an animation to the menu. Not ideal, but it seems to fix the issue.